network-api/networkapi/mozfest/blocks/mixed_content_block.py

```diff
@@ -0,0 +1,63 @@
+from django.forms.utils import ErrorList
+from wagtail import blocks
+from wagtail.blocks import struct_block
+from wagtail.images import blocks as image_blocks
+
+from networkapi.wagtailpages.pagemodels.customblocks import listing as listing_blocks
+
+
+class VideoBlock(blocks.StructBlock):
+    url = blocks.CharBlock(
```

Comment:
Is it on purpose that this is not using a `URLBlock`?

Reply:
Or an `EmbedBlock`?

Reply:
I went off the other video_block I found, but this should be a URL block at least yea, I've changed it to that. Chris is doing the FE at the moment so I'm unsure as to whether we want this to be and embed of just a URL value tbh